packages/shared_preferences/shared_preferences_macos/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: shared_preferences_macos
+description: macOS implementation of the shared_preferences plugin.
+version: 0.0.1
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/shared_preferences/shared_preferences_macos
+
+flutter:
+  plugin:
+    platforms:
+      macos:
+        pluginClass: SharedPreferencesPlugin
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
```

Comment:
This should probably have Flutter SDK bounds too?

Reply:
Done